wazo_confd/helpers/destination.py

```diff
@@ -272,7 +272,7 @@ class NoneDestinationSchema(BaseDestinationSchema):
 class OutcallDestinationSchema(BaseDestinationSchema):
     outcall_id = fields.Integer(attribute='actionarg1', required=True)
     exten = fields.String(
-        validate=(Predicate('isdigit'), Length(max=255)),
+        validate=Regexp(EXTEN_REGEX),
```

Comment:
For now, I'll revert this commit:
1. We don't have real reason to change it
2. I don't know impact to allow `#*` in this field, and I'm quit lazy to spend time to investigate if we need it or not.
  * No objection to make this investigation if needed
3. This change behavior of the API by not allowing to set empty string
  * This is a good validation, but we should add migration script to avoid having empty string in database and make wrong logic based on this new validation

Reply:
All good, thank you for your feedback !